src/solvers/LevenbergMarquardt.jl

```diff
@@ -208,6 +208,19 @@ function initialize_solver!(
     return lms
 end
 
+function default_lm_lin_solve!(sk, JJ, grad_f_c)
```

Comment:
could you maybe add a small doc string here as well so that the doc string above could `@ref` to this?

Alternatively when mentioning the default describe short what it does (cholesky or \).

Reply:
Sure, I've added a docstring.

Reply:
Ah and a

```@docs
Manopt.default_lm_lin_solve!
```
block or such is added in an internals section for LM in the docs.

Reply:
OK, done.